.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v5.0.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files

```

Comment:
@tfardet I think that's the part which impacted svgs and the like, nothing to worry too much